build/prbuild.yml

```diff
@@ -56,6 +56,14 @@ jobs:
       configuration: release
       rerunFailedTests: true
 
+  - task: DotNetCoreCLI@2
```

Comment:
Should we add something like "run-dotnetcore-tests" as a template stored in a separate file? This would mean a future change to the task's arguments would only need to touch one line, not five. debug/release could be passed in as a parameter, [like with the ai-win UI test job](https://github.com/microsoft/accessibility-insights-windows/blob/db0c9b0435f8694b3841bcf61efdfeb3899b1112/build/prbuild.yml#L21)

Reply:
Let's consider this as a separate YAML refactoring PR. The objective is this PR is to ensure that our .NET Core unit tests are running correctly